README.md

```diff
@@ -100,41 +121,50 @@ Each line is a single event collected by Tracee, with the following information:
+
+You can filter by most of the visible fields on a Tracee event. For example to trace only events related to user ID 1000, use `--trace uid=1000`.  
+You can combine trace expressions into more complex criteria. For example, to trace only events related to user ID 1000, which come from process ID 1234, use `--trace uid=1000 --trace pid=1234`.  
+
+A special `pid` value is `new` which let's you trace all newly created processes (that were created after Tracee started tracing).  
+Tracee lets you easily trace events that originate in containers using `--trace container` or only new containers (that was created after Tracee started) using `--trace container=new`.
+
+Event metadata can be used in trace expression as well. For example, to trace only `openat` syscalls, use `--trace event:openat`. But you can also filter on a specific argument of the event, e.g `--trace openat.pathname=/bin/ls` which will show only `openat` syscalls that operate on the file `/bin/ls`.
+
+A useful trace mode is the `--trace follow` which, if specified, will trace not only processes that match the given trace expressions, but also their child processes.
+For example, the following will trace all the events that originate from zsh shell, including all of the processes that it will spawn: `--trace command=zsh --follow`.
 
-1. UTS_NAME - uts namespace name. As there is no container id object in the kernel, and docker/k8s will usually set this to the container id, we use this field to distinguish between containers.
-2. MNT_NS - mount namespace inode number.
-3. PID_NS - pid namespace inode number. In order to know if there are different containers in the same pid namespace (e.g. in a k8s pod), it is possible to check this value
-4. PPID - parent pid of the calling process
+For a complete list of trace options, run `--trace help`.
 
+### Capturing artifacts
 
-### Configuration flags
+Tracee has a unique feature that lets you capture interesting artifacts from running applications, using the `--capture` flag.
 
-- Use `--help` to see a full description of all options.
-Here are a few commonly useful flags:
-- `--trace` Sets the trace mode. For more information see [Trace Mode Configuration](#Trace-Mode-Configuration) below
-- `--event` allows you to specify a specific event to trace. You can use this flag multiple times, for example `--event execve --event openat`.
-- `--list` lists the events available for tracing, which you can provide to the `--event` flag.
-- `--output` lets you control the output format, for example `--output json` will output as JSON lines instead of table.
-- `--capture` capture artifacts that were written, executed or found suspicious, and save them to the output directory. Possible values are: 'write'/'exec'/'mem'/'all'
+All captured artifacts are saved in Tracee's "output directory" which can be configured using `--capture dir:/path/to/dir`.
 
-### Trace Mode Configuration
+Tracee can capture the following types of IoCs:
```

Comment:
What is IoC?

Reply:
Indicator of compromise.

Reply:
Roger - we should put that here as it's the first usage within this document.
```suggestion
Tracee can capture the following types of Indicator of Compromise (IoC):
```